bl-service/src/test/java/com/ayeeti/blservice/user/UserServiceTest.java

```diff
@@ -0,0 +1,55 @@
+package com.ayeeti.blservice.user;
+
+import com.ayeeti.blservice.location.requests.LocationRequest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Set;
+
+@SpringBootTest
```

Comment:
Again you are writing tests. Why didn't you write a LocationController? Why is the task not completed?

Reply:
I didn't see the task. I thought I was doing the create user at a given location. But now I have added the Location controller.